Software_Carbon_Intensity/Appendix_A_Further_Information_on_Reporting_Requirements.md

```diff
@@ -22,6 +22,8 @@ The following list contains the REQUIRED and OPTIONAL data elements to be report
 | Software or Product Version | metadata/entity-version | MUST | Text | |
 | SCI Specification Version | metadata/sci-version | MUST | Numeric | The version of the SCI specification against which this report is being made. |
 | Date of Calculation | metadata/calculation-date | MUST | Date | Following a format described in [RFC3339], [a subset](https://ijmacd.github.io/rfc3339-iso8601/) of ISO 8601 |
+| Start date of the workload | metadata/workload-start-time | MAY | Timestamp | Following a format described in [RFC3339], [a subset](https://ijmacd.github.io/rfc3339-iso8601/) of ISO 8601 |
+| Processing period for calculation | metadata/calculation-period | MAY | Numeric | Time taken in seconds to complete the process defined by `R`.  |
```

Comment:
Should the time be in `ms` since some tasks / processes might be much more granular than taking seconds?

Reply:
I've updated to use ms instead of sec.

I thought it may be suitable to use sec here because the task period could be long (e.g. ML job). However we should also consider short-term task (e.g. transaction). So I accepted the suggestion from @atg-abhishek .